src/lang.js

```diff
@@ -8,6 +8,40 @@
    * @words expect array
    *
    */
+
+  let setLangType = function (LangType) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['setLangType' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)